docs/tutorials/upgrade-a-chain/sudo-upgrade.md

```diff
@@ -0,0 +1,163 @@
+from the `Root` origin. The `sudo_unchecked_weight` function is what will be used to invoke the
+runtime upgrade in this section of this tutorial; in the next section, the Scheduler pallet will be
+used to manage the resources consumed by the `set_code` function.
+
+## Prepare an Upgraded Runtime
+
+Because the template node doesn't come with the Scheduler pallet included in its runtime, the first
+runtime upgrade performed in this tutorial will add that pallet. First, add the Scheduler pallet as
+a dependency in the template node's `runtime/Cargo.toml` file.
+
+```toml
+[dependencies.pallet-scheduler]
+default-features = false
+git = 'https://github.com/paritytech/substrate.git'
+tag = 'v2.0.0-rc6'
+version = '2.0.0-rc6'
+
+#--snip--
+
+[features]
+default = ['std']
+std = [
+    #--snip--
+    'pallet-scheduler/std',
+    #--snip--
+]
+```
+
+Next, add the pallet to the runtime by updating `runtime/src/lib.rs`.
+
+```rust
+// Define the types required by the Scheduler pallet.
+parameter_types! {
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * MaximumBlockWeight::get();
+}
+
+// Configure the runtime's implementation of the Scheduler pallet.
+impl pallet_scheduler::Trait for Runtime {
+	type Event = Event;
+	type Origin = Origin;
+	type PalletsOrigin = OriginCaller;
+	type Call = Call;
+	type MaximumWeight = MaximumSchedulerWeight;
+	type ScheduleOrigin = frame_system::EnsureRoot<AccountId>;
+	type WeightInfo = ();
+}
+
+// Add the Scheduler pallet inside the construct_runtime! macro.
+construct_runtime!(
+	pub enum Runtime where
+		Block = Block,
+		NodeBlock = opaque::Block,
+		UncheckedExtrinsic = UncheckedExtrinsic
+	{
+		/*** snip ***/
+		Scheduler: pallet_scheduler::{Module, Call, Storage, Event<T>},
+	}
+);
+```
+
+The final step to preparing an upgraded FRAME runtime is to increment its
+[`spec_version`](https://substrate.dev/rustdocs/v2.0.0-rc5/sp_version/struct.RuntimeVersion.html#structfield.spec_version),
+which is a member of
+[the `RuntimeVersion` struct](https://substrate.dev/rustdocs/v2.0.0-rc5/sp_version/struct.RuntimeVersion.html)
+that is defined in `runtime/src/lib.rs`.
+
+```rust
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("node-template"),
+	impl_name: create_runtime_str!("node-template"),
+	authoring_version: 1,
+	spec_version: 2,  // Update this value.
+	impl_version: 1,
+	apis: RUNTIME_API_VERSIONS,
+	transaction_version: 1,
+};
+```
+
+Build the upgraded runtime.
+
+```shell
+cargo build --release -p node-template-runtime
+```
+
+## Upgrade the Runtime
+
+Use this link to open the Polkadot JS Apps UI and automatically configure the UI to connect to the
+local node: https://polkadot.js.org/apps/#/extrinsics?rpc=ws://127.0.0.1:9944. Use Alice's account
+to invoke the `sudo_unchecked_weight` function and use the `set_code` function from the System
+pallet as its parameter. In order to supply the build artifact that was generated by the previous
```

Comment:
```suggestion
pallet as its parameter. The reason we can use Alice account to make the sudo call is because we have added Alice private key in Sudo keystore when we run the node in development mode (started with `--dev` flag). In order to supply the build artifact that was generated by the previous
```